README.md

```diff
@@ -101,6 +146,18 @@ bind::zone::primary { 'example.com':
   content       => epp("profile/dynamic-zone-template.epp", $params),
 }
 ```
+Or with hiera
+```puppet
+bind::keys:
+  nsupdate:
+    secret: TopSecret
+    keyfile: /etc/bind/nsupdate.key
+
+bind::zone::primaries:
+  example.com:
+    update_policy: grant nsupdate zonesub any
+    content: 'epp("profile/dynamic-zone-template.epp", $params)'
```

Comment:
This won't work. The content of the zone file will be the given string and not the content of the template.